lib/hanami/logger.rb

```diff
@@ -214,17 +215,33 @@ def _format(hash)
 
       # @api private
       def _format_error(result, hash)
-        result << " #{hash[:error]}:" if hash.key?(:error)
-        result << " #{hash[:message]}#{NEW_LINE}"
+        error_message = if tty? Hanami::Utils::ShellCode.colorize(
+          [ hash[:error], hash[:message] ].compact.join(": "),
+          color: :red,
+          )
+        else
+          [ hash[:error], hash[:message] ].compact.join(": ")
+        end
+        result << [" ", error_message, NEW_LINE].join
         if hash.key?(:backtrace)
           hash[:backtrace].each do |line|
-            result << "from #{line}#{NEW_LINE}"
+            if tty?
+              result << Hanami::Utils::ShellCode.colorize("from #{line}#{NEW_LINE}", color: :yellow)
+            else
+              result << "from #{line}#{NEW_LINE}"
```

Comment:
Just a idea, but what about `colorise` method (sorry, wired name) for wrap this tty condition? Something like this:
```ruby
private

def colorise(text, color:)
  if tty?
    Hanami::Utils::ShellCode.colorize(text, color: color)
  else
    text
  end
end
```

And call it like this:
```ruby
colorise("from #{line}#{NEW_LINE}", color: :yellow)
colorise([ hash[:error], hash[:message] ].compact.join(": "), color: :red)
```

WDYT?